agent/ecscni/plugin.go

```diff
@@ -113,15 +115,24 @@ func (client *cniClient) setupNS(cfg *Config) (*current.Result, error) {
 	os.Setenv("ECS_CNI_LOGLEVEL", logger.GetLevel())
```

Comment:
when vpc-branch-eni plugin is used, similar to here, you will need to specify the log level of vpc-branch-eni via "VPC_CNI_LOG_LEVEL"; you also need to specify "VPC_CNI_LOG_FILE" to be "/log/vpc-branch-eni.log" since the default log path used by the plugin is "/var/log/vpc-branch-eni.log" which is not under the log path that's mounted to the agent (which is [this](https://github.com/aws/amazon-ecs-init/blob/master/ecs-init/docker/docker.go#L32))

Reply:
same in cleanup